src/mixins/socket.js

```diff
@@ -241,7 +241,7 @@ export default {
 
             socket.on("disconnect", () => {
                 console.log("disconnect");
-                this.connectionErrorMsg = "Lost connection to the socket server. Reconnecting...";
+                this.connectionErrorMsg = `${this.$t("Lost connection to the socket server. Reconnecting...")}`;
```

Comment:
No need to create additional errfort for our translatiors, right?

```suggestion
                this.connectionErrorMsg = `${this.$t("Lost connection to the socket server.")} ${this.$t("Reconnecting...")}`;
```

Reply:
Therefore, the key must also be changed. "Lost connection to the socket server." to "Cannot connect to the socket server" to match the existing key ?

Reply:
yes, missed that one. The key should also be added to the `en.json`